ormconfig.json

```diff
@@ -0,0 +1,14 @@
+{
+  "type": "postgres",
+  "host": "localhost",
+  "port": 5432,
+  "username": "postgres",
+  "password": "1234",
+  "database": "3d-ingestion",
+  "entities": ["src/request/models/*.ts"],
+  "migrationsTableName": "requests_migration_table",
+  "migrations": ["db/migration/*.ts"],
+  "cli": {
+    "migrationsDir": "db/migration"
+  }
+}
```

Comment:
The fields that are configured in the config should be retrieved from the config module and not hardcoded.